src/Tizen.NUI/src/internal/Interop/Interop.AccessibilityManager.cs

```diff
@@ -89,6 +89,12 @@ internal static partial class AccessibilityManager
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_AccessibilityManager_GetReadPosition")]
             public static extern global::System.IntPtr AccessibilityManager_GetReadPosition(global::System.Runtime.InteropServices.HandleRef jarg1);
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_AccessibilityManager_EnableAccessibility")]
+            public static extern void AccessibilityManager_EnableAccessibility(global::System.Runtime.InteropServices.HandleRef jarg1, bool jarg2);
+
```

Comment:
please remove "\_" in NUI API naming. I think AccessiblityManager is repeated.
so how about this ?
AccessibilityManager_EnableAccessibility => EnableAccessibility
I can see other legacy code uses "_" here, so please change these namimgs for next time.

Reply:
Thank you for your review.
I updated it to remove the duplicated prefix. :)